mcclatchy/design
# cleaning up card and digest shortcodes

### What does this PR do?

It makes the shortcodes more dynamic and incorporates static versions of some public section feeds as data sources for templates.

1. Digests have a new feed property to pick the feed of choice (default "news")
2. Cards auto-increment themselves as they are added to decks (max 20 for now)

In addition to this cleanup, this also adds some JS functionality to append the `data-layout` attribute to the `<html>` tag on hashchange. It also adds some CSS to show Taboola regions so we can talk those through with stakeholders.

### Steps to test

Please take a look at the source code as well and hit me with any questions or concerns over the approach.

1. Check out the new branch and fire up the Hugo server.
2. Take a look at all decks, but specifically the homepage and section versions. 
3. On the hompage deck, add `#taboola` to the url to see the overlay.

### Here's what the overlay should look like.
![Screenshot 2023-08-16 5 14 07 PM](https://github.com/mcclatchy/design/assets/198070/d0be839e-23a2-4c19-a630-33836617d4b3)
